internal/api/v1/docs/namespace.go

```diff
@@ -0,0 +1,66 @@
+package docs
+
+import (
+	"github.com/epinio/epinio/pkg/api/core/v1/models"
+)
+
+//go:generate swagger generate spec
+
+// swagger:route GET /namespaces namespace Namespaces
+// Return map of all controlled namespaces.
+// responses:
+//   200: NamespacesResponse
+
+// swagger:response NamespacesResponse
+type NamespacesResponse struct {
+	// in: body
+	Body map[string]models.NamespaceList
+}
+
+// swagger:route POST /namespaces/{namespace} namespace NamespaceCreate
+// Create a new named `namespace`.
+// responses:
+//   200: NamespaceCreateResponse
+
+// swagger:parameters NamespaceCreate
+type NamespaceCreateParam struct {
+	Namespace models.NamespaceCreateRequest
```

Comment:
I'm not sure about this one: should it be `string` instead of `models.NamespaceCreateRequest`?

Reply:
It should be `string`, and there should be a
```
// in: body
Body models.NamespaceCreateRequest
```
for the POST body.